content/en/data_streams/dotnet.md

```diff
@@ -22,7 +22,18 @@ For example:
 environment:
   - DD_DATA_STREAMS_ENABLED: "true"
 ```
+### Libraries Supported
+Data Streams Monitoring supports the [confluent-kafka library][4].
+
+### Monitoring SQS Pipelines
+Data Streams Monitoring uses one [message attribute][5] to track a message's path through an SQS queue. As Amazon SQS has a maximum limit of 10 message attributes allowed per message, all messages streamed through the data pipelines must have 9 or less message attributes set, allowing the remaining attribute for Data Streams Monitoring.
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
```

Comment:
```suggestion
```

Reply:
I think we can remove this for now as there aren't any Further Reading links listed in the Markdown frontmatter, which means the Further Reading heading will be displayed without any links listed below it.